playground/tests/test_alive.py

```diff
@@ -0,0 +1,5 @@
+import pytest_container
+
+
+def test_alive(auto_container: pytest_container.container.ContainerData, host):
```

Comment:
You need to define which container images this test acts upon by setting the file global variable `CONTAINER_IMAGES`. Setting it in `conftest.py` is not sufficient.

Reply:
Thank you for your feedback. Should I also duplicate the image definition?

Reply:
No, just import `TW` from `conftest.py`.